### PR TITLE
Ensure bazel is really brought down (ONLY on bazel-release target)

### PR DESCRIPTION
**Pull request by dims**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Looks like `bazel stop` does not really always stop the processes. Let us do this explicitly

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by BenTheElder — COMMENTED** [review]
Do we know if this works? 🙃

**Comment by dims** [comment]
`pkill ^bazel` does seem to work :)

File: build/root/Makefile [hunk 1]
```diff
@@ -624,4 +624,5 @@ else
 bazel-release:
 	bazel build //build/release-tars
 	bazel shutdown
+	pkill ^bazel || true
```
Comment (aojea) [inline build/root/Makefile]: should we kill all the java processes that are bazel leftovers?

https://github.com/bazelbuild/bazel/issues/1868
https://github.com/bazelbuild/bazel/issues/4505#issuecomment-406312017

Comment (BenTheElder) [reply build/root/Makefile]: > bazel shutdown stops workers only from the current workspace.

we only have one workspace. though it seems shutdown isn't cutting it ...?

**Comment by BenTheElder** [comment]
bah
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/87929/pull-kubernetes-e2e-gce/1225962193816653824/

[k8s.io] Probing container should *not* be restarted with a non-local
redirect http liveness probe expand_less 4m22s
test/e2e/common/container_probe.go:248
Feb  8 02:39:22.595: pod
container-probe-5056/liveness-d256311c-7911-4bc3-b073-c644f0d05475 -
expected number of restarts: 0, found restarts: 1
test/e2e/common/container_probe.go:463

that's flaky due to other sources of restarts I bet :/
I don't think this is the only test doing this either

/retest

**Pull request merged**